TACC/protx-dashboard
# Task/COOKS-310: complete saga and update selectors

## Overview: ##
Complete new saga, update selectors, minor adjustments to chart section (e.g. instructions & new header)

## Related Jira tickets: ##

* [COOKS-123](https://jira.tacc.utexas.edu/browse/COOKS-123)

## Summary of Changes: ##

- New redux saga and reducer for getting statewide analytics charts
- Updated selectors per Juliana's design document
- Added header and updated instruction in the chart section

## Testing Steps: ##
1. Go to https://cep.test/protx/dash/analytics
2. Test the new dropdown menu for Projections and ensure that selecting each option ('Relative Risk' or 'Predicted Number of Cases') causes the intended chart to display.
3. Note any desired changes to chart header and/or instructions.

## UI Photos:
**Relative Risk Chart Selected**
![Screen Shot 2022-10-19 at 12 22 29 PM](https://user-images.githubusercontent.com/79928051/196761484-74a9ea70-af1d-4dd8-a3c3-d6bb465f04bf.png)

**Predicted Number of Cases Chart Selected**
![Screen Shot 2022-10-19 at 12 23 32 PM](https://user-images.githubusercontent.com/79928051/196761991-71b8b633-0e09-4053-a151-99d4103cafd9.png)

## Notes: ##
